### PR TITLE
pdp description banner

### PR DESCRIPTION
**Pull request by Aleena-4**

### CODE REVIEW

File: src/libs/ui/src/molecules/AccordianItem/index.tsx [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
 }
 
-const AccordionItem: React.FC<AccordionItemProps> = ({
+export interface AccordionItemRef {
+  isActive: boolean;
+ 
+}
+
+const AccordionItem = forwardRef<AccordionItemRef, AccordionItemProps>(({
   title,
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/AccordianItem/index.tsx]: Please remove unused code forwardRef

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

**Comment by Aleena-4** [comment]
Screenshot 2024-10-17 191116
Screenshot 2024-10-17 191126
[
Screenshot 2024-10-17 190920
](url)

File: src/libs/ui/src/molecules/VaidationModal/index.tsx [hunk 2]
```diff
@@ -1,22 +1,70 @@
...
+}
 
+const ValidationModal: React.FC<ValidationModalProps> = ({ password }) => {
+  const passwordRequirements = [
+    { label: 'At least 8 characters', regex: /.{8,}/ },
+    { label: 'Contains an uppercase letter', regex: /[A-Z]/ },
+    { label: 'Contains a lowercase letter', regex: /[a-z]/ },
+    { label: 'Contains a number', regex: /[0-9]/ },
+    { label: 'Contains a special symbol', regex: /[!@#$%^&*(),.?":{}|<>]/ },
+  ];
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/VaidationModal/index.tsx]: move this constant variable outside of the component passwordRequirements

**Comment by Aleena-4** [comment]
Screenshot 2024-10-17 190920
Screenshot 2024-10-17 191116
Screenshot 2024-10-17 191126
Screenshot 2024-10-17 191135

**Comment by Aleena-4** [comment]
Updated the images

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

File: src/libs/ui/src/organisms/RelatedProducts/index.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <div key={relatedProducts.id} className="mt-1">
+
+              <Product className='!h-[40px] text-center !py-[0.5rem]' product={relatedProducts} modalSetToggle={function (): void {
+                throw new Error('Function not implemented.');
+              }} />
+            </div>
+          ))}
+        </Slider>
+      </div>
+      <div className='pt-20 lg:px-4 '>
```
Comment (Vishnu9901) [inline src/libs/ui/src/organisms/RelatedProducts/index.tsx]: Remove MarketingBannerTwo sesction from Releated Products component

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

**Comment by Aleena-4** [comment]
Closed due to merge conflicts

**Pull request closed without merging**